Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/storage/entity/TraceLocationConverters.kt

```diff
@@ -1,13 +1,14 @@
 package de.rki.coronawarnapp.eventregistration.storage.entity
 
 import androidx.room.TypeConverter
-import de.rki.coronawarnapp.eventregistration.events.TraceLocation
+import de.rki.coronawarnapp.server.protocols.internal.pt.TraceLocationOuterClass
 
 class TraceLocationConverters {
 
     @TypeConverter
-    fun toTraceLocationType(value: Int) = enumValues<TraceLocation.Type>().single { it.value == value }
+    fun toTraceLocationType(value: Int): TraceLocationOuterClass.TraceLocationType =
+        TraceLocationOuterClass.TraceLocationType.forNumber(value)!!
```

Comment:
Would this crash here if the mapping is missing?

Reply:
Yes, it would crash if the user has a value in his database that we can't map to an enum value. 

However, this can only happen if we change the enum values for a new app version. In this case, we also would need to write a database migration in which we replace the invalid values with new values. 

I discussed this with @d4rken and he also recommended the crash (instead of e.g. returning a fallback value) as this situation is unexpected. This way we also notice this issue faster.